### ISSUE TITLE
The PasswordAuthStrategy defaults cannot be used in production

### ISSUE THREAD

**Issue by MadeByMike** [keystonejs/keystone-5#1456]
When you add a PasswordAuthStrategy you get a warning in the console:

```
Connecting to databaseWarning: connect.session() MemoryStore is not designed for a production environment, as it will leak memory, and will not scale past a single process.
```

This is because of express session: https://github.com/expressjs/session/blob/master/README.md#compatible-session-stores 

Looks like there are a bunch of better defaults we could use? Are there good docs on how to customise this?

**Comment by gautamsi** [comment]
This is what I did to use mongodb based Store

```js
const session = require('express-session');
const MongoStore = require('connect-mongo')(session);
const sessionStore = new MongoStore({url: CONNECT_TO});
// ......

module.exports = {
  keystone,
  apps: [
    new GraphQLApp({ sessionStore }),
    new StaticApp({ .... }),
    adminApp,
    new NextApp({ .... }),
  ],
};
```

**Comment by MadeByMike** [comment]
Thanks @gautamsi, I'll make sure that ends up in the docs. @jesstelford What do you think about making this the default session store?? https://www.npmjs.com/package/memorystore

I want to make our demos and defaults as close to ready to deploy  as possible.

**Comment by MadeByMike** [comment]
Seems like it would be a matter of adding it here: https://github.com/keystonejs/keystone-5/blob/5e56bc4d4754ca4242b8f1d4cb6b819873a3254e/packages/session/lib/session.js when `sessionStore` is undefined?

**Comment by jesstelford** [comment]
I'd prefer to leave the warning there instead of moving to memoryStore.

The memoryStore only works on single instance server environments,. Everywhere else it'll intermittently break as your requests are routed to different servers which have different values (or none at all in serverless cases) in their memory. That would be a silent and hard to track down issue. At least this way it's noisy about not being safe.

**Comment by MadeByMike** [comment]
Sure! Are there any other alternative types of stores we could use?

**Comment by MadeByMike** [comment]
Closed and replaced with #1511

**Comment by leobar37** [comment]
```
const sessionExpress = require("express-session");
const knexSesionStore = require("connect-session-knex")(sessionExpress);


 const app = new Keystone({
      secureCookies: false,
      adapter: this.bdAdapter(),
      sessionStore: new knexSesionStore({
        tablename: "sesions",
        knex: knex(configKnex),
      })
 });,
``
